modules/statistics/statistics.js

```diff
@@ -318,8 +318,10 @@ Statistics.prototype.startCallStats = function(tpc, remoteUserID) {
         if (!CallStats.initBackend({
             callStatsID: this.options.callStatsID,
             callStatsSecret: this.options.callStatsSecret,
-            userName,
-            aliasName: this.options.callStatsAliasName
+            userName: this.options.swapUserNameAndAlias
+                ? this.options.callStatsAliasName : userName,
+            aliasName: this.options.swapUserNameAndAlias
+                ? userName : this.options.callStatsAliasName
```

Comment:
What benefit does the swap provide? I'm guessing for easier searching/filtering but I'm unsure.

Reply:
Yes, so till now we are sending callstatsUsername as alias and username is the endpoint ID which is taken from the jid and changes every call. To view aliases you need to be admin.
With this option that I'm adding we send callstatsUsername as username, and eventually the display name (behind a flag) as an alias.
As callstatsUsername is not changing between calls in callstats you can search and find all calls for this user and only admins can see its real name if this option is enabled.